wiki/Help_centre/Upgrading_to_lazer/en.md

```diff
@@ -0,0 +1,235 @@
+
+Scores will already have performance points calculated (you can see this in the "recently played" section of your profile), but it will not contribute to the total value yet.
+
+#### Does lazer use ScoreV2?
+
+Lazer currently uses a new experimental score implementation which is similar to ScoreV2 but not the same. This is still in development and we are looking for more feedback on how it feels in various contexts (solo play, leaderboards, tournaments etc.)
+
+#### I prefer the classic scoring display, where scores get really big. Why was this changed?
+
+You can actually change the "Score display mode" in settings to "Classic" to get back the explosive style of scoring game-wide! It won't be a perfect match, but will give you the same feel of classic scoring and be applied everywhere you'd expect it to be.
+
+#### If I set a score on lazer, will it remain forever?
+
+While we will try to preserve as many scores as possible, we **offer no guarantee that scores will remain indefinitely**. At any point we may choose to wipe a subset or all scores in order to preserve game balance.
+
+#### Will scores set on stable eventually show in lazer?
+
+Yes. Once we finish balancing the combination of lazer and stable scores, both will be visible.
+
+#### Will all mods be ranked?
+
+For now, scores of all mod combinations appear on leaderboards. Whether scores will give performance points with all mods (and if they do, whether there will be a bonus or penalty applied) is still in discussion.
+
+#### I don't like the new gameplay mechanics. Can I restore the old gameplay mechanics like on stable?
+
+Please try applying the "classic" mod, which will restore much of the old behaviour that you are used to. Also make sure to check the settings offered by classic mod, as it will let you further customise your experience and also understand what changes are being applied (as they are all listed there).
+
+### Skinning and UI
+
+#### Something is behaving differently to stable and I don't like it!
+
+Please run the setup wizard at the top of settings and go through the settings on the `Behaviour` screen. A lot of the common settings which have defaults changed are listed here. There's also a single button you can press to apply the old behaviours as a starting point for your lazer journey.
+
+#### Will old skins eventually work in song select and results screens?
+
+We'll do our best to bring back as much of this as we can without blocking new functionality. This will come later on.
+
+#### Can I use my skin cursor in the menus as well?
+
+We will likely bring back support for this in the future due to popular demand.
+
+### Performance
+
+#### Why can't I run at unlimited FPS?
+
+There is no reason to run at high frame rates. Lazer employs various new technologies to ensure the lowest latency is achievable without requiring high frame rates. This will continue to improve going forward as we still have a few improvements left to implement.
```

Comment:
Not sure if this isn't muddying the waters but this statement without any qualification kinda caught me off guard:

```suggestion
Above a certain threshold there is no reason to run at higher frame rates. Lazer employs various new technologies to ensure the lowest latency is achievable without requiring high frame rates. This will continue to improve going forward as we still have a few improvements left to implement.
```

Reply:
i'd word it as something akin to: "Despite common belief, there is no reason to run at very high frame rates in Lazer, as new technologies are employed to ensure the... " along with some emphatic promise that this is actually the case and people should try it out first

Reply:
sounds a bit wonky because it's not like people believe that there is a reason to run at very high frame rates *in lazer specifically*